docs/source/examples/BSTLD.rst

```diff
@@ -21,22 +21,19 @@ See `this page <https://www.graviti.cn/open-datasets/BSTLD>`_  for more details
 
```

Comment:
Maybe you can add some attribute to the `code` in commit message.
Like `the example code` or `the demo code` to make it more clear
At least you need `the` before it: `the code`

Actually you haven't replace the code. What you did is use another way to display the same code.
You need to reconsider your message~